src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -67,6 +66,32 @@ public static Entity getEntityFromDatastore(String entityName, String entityProp
     return entity;
   }
 
+     /**
+    * This function takes in a list of cookies and matches a sessionID cookie
+    * with a specific name/value pair to a user entity with the same name/value
+    * pair as one of its properties. This user entity is then returned. 
+    * 
+    * <p>
+    * 
+    * @param  cookies    an array of cookies (usually all cookies on front end)
+    * @param  datastore  the database where entities are stored
+    * @return            a single entity that has the cookie name/value pair as a property
+    */
+ 
+  public static Entity getUserFromCookie(Cookie cookies[], DatastoreService datastore){
+      if(datastore == null){
+          log.severe("Error in function getUserFromCookie(): Datastore passed in was null");
+          return null;
+      }
+    String sessionIDName = "SessionID";
```

Comment:
If you make this a class constant (with `static final String sessionIdCookieName = "SessionID";` at the class level), then you can refer to that constant in your unit tests and other cookie-related code without the risk of misspelling "SessionID". (Which sounds silly but is the kind of thing that happens a lot and is hard to debug.)

Reply:
Good, you did. Please also replace "SessionID" in the unit test file